test/integration/targets/eos_l2_interfaces/tests/cli/reset_config.yml

```diff
@@ -21,7 +21,8 @@
       - name: Ethernet1
         access:
           vlan: 20
-      - name: Ethernet2
+      - mode: trunk
+        name: Ethernet2
```

Comment:
cli_config has 'switchport mode trunk' under Ethernet2

Reply:
I didn't get the issue here, can u give me n example. Thanks

Reply:
I see 'switchport mode trunk' for Ethernet2 under cli_config. But the expected_config, does not have that configuration for Ethernet2. Am I missing something here? Is the assertion successful?

Reply:
command [switchport mode trunk](https://github.com/ansible/ansible/pull/67456/files#diff-e64d12f3b58aa06edbc84ae3c3ec8a8dR12) is getting configured and as a result [mode:trunk](https://github.com/ansible/ansible/pull/67456/files#diff-e64d12f3b58aa06edbc84ae3c3ec8a8dR24) is configured in expected_config, isn't that expected?